www/pages/sell/new.tsx

```diff
@@ -512,7 +512,12 @@ function CreateOrderButton(props: {
       )}
       {errorMessage && (
         <>
-          <button className="cursor-not-allowed mt-4 bg-red-500 text-white px-4 py-2 w-full flex justify-center font-bold rounded">
+          <button
+            className="cursor-not-allowed mt-4 bg-red-500 text-white px-4 py-2 w-full flex justify-center font-bold rounded"
+            onClick={() => {
+              setErrorMessage('');
+            }}
+          >
```

Comment:
Resetting the error message to `''` is the perfect way to implement retry here. Do you think it will be obvious what clicking the error button does tho?

Maybe this should be implemented as a second retry button bellow the errored out button? (only in the if `errorMessage` section). Thoughts?

Reply:
sure!